forch/authenticator.py

```diff
@@ -57,14 +66,23 @@ def process_auth_result(self):
             auth_example = dict_proto(auth_obj, AuthResult)
             sys.stdout.write(str(proto_dict(auth_example)) + '\n')
 
-    def do_mab_request(self, _args):
+    def do_mab_request(self, src_mac, port_id):
         """Initiate MAB request"""
-        Socket = collections.namedtuple('Socket', 'listen_ip, listen_port, server_ip, server_port')
-        socket_info = Socket('0.0.0.0', 0, _args.server_ip, _args.server_port)
-        radius_query = RadiusQuery(socket_info, _args.radius_secret)
-        LOGGER.info('sending MAB request')
-        radius_query.send_mab_request(_args.src_mac, _args.port_id)
-        radius_query.receive_radius_messages()
+        LOGGER.info('sending MAB request for %s', src_mac)
+        self.radius_query.send_mab_request(src_mac, port_id)
+
+    def process_device_placement(self, src_mac, device_placement):
+        """Process device placement info and initiate mab query"""
+        if not self.radius_query:
+            LOGGER.warning("RADIUS query module not setup. Ignoring auth request for %s", src_mac)
```

Comment:
this should error out -- this method shouldn't be called if module not setup

Reply:
I am removing the check altogether, so it errors out

Reply:
done.